src/app/ubs/ubs/components/ubs-confirm-page/ubs-confirm-page.component.ts

```diff
@@ -59,20 +59,20 @@ export class UbsConfirmPageComponent implements OnInit, OnDestroy {
         this.orderService
           .getUbsOrderStatus()
           .pipe(takeUntil(this.destroy$))
-          .subscribe(
-            (response) => {
+          .subscribe({
+            next: (response) => {
```

Comment:
There are a few potential issues and improvements that can enhance the clarity in `getUbsOrderStatus`:
`orderService.getUbsOrderStatus(paymentOrderId)): Observable<FondyPaymentResponse>`.
and The signature '(error: any): Observable<never>' of 'throwError' is deprecated.
You might want to add user feedback, such as an error message displayed on the UI better use `matSnackBar.openSnackBar`